build/buildconstants/params_shared_vals.go

```diff
@@ -103,3 +103,8 @@ const InitialBaseFee int64 = 100e6
 const MinimumBaseFee int64 = 100
 const PackingEfficiencyNum int64 = 4
 const PackingEfficiencyDenom int64 = 5
+
```

Comment:
change here should dedupe with https://github.com/filecoin-project/lotus/pull/13034